docs/rules/array-bracket-newline.md

```diff
@@ -0,0 +1,237 @@
+# enforce line breaks after opening and before closing array brackets (array-bracket-newline)
+
+A number of style guides require or disallow line breaks inside of array brackets.
+
+## Rule Details
+
+This rule enforces line breaks after opening and before closing array brackets.
+
+## Options
+
+This rule has either a string option:
+
+* `"always"` requires line breaks inside braces
+* `"never"` disallows line breaks inside braces
+
+Or an object option (Requires line breaks if any of properties is satisfied. Otherwise, disallows line breaks):
+
+* `"multiline": true` (default) requires line breaks if there are line breaks inside elements or between elements. If this is false, this condition is disabled.
+* `"minItems": null` requires line breaks if the number of elements is at least the given integer. If this is 0, this condition will act the same as the option `"always"`.
```

Comment:
Explicitly stating here what the default is and what effect`minItems` will have when it is `null` will make this more clear.